src/features/query-bar/components/filter/ValueField.js

```diff
@@ -58,6 +60,9 @@ const ValueField = ({ onChange, selectedValues, valueOptions, valueType }: Props
             return (
                 <div className="filter-dropdown-text-field-container">
                     <TextInput
+                        className={className}
```

Comment:
Why is className set on an inner element? This effectively bars a future solution where clients can select the outermost <div/>.

Reply:
Classname is set here since there are unique styles that are required for each input field to make it look correct in the Grid.

Reply:
Is it possible to solve what you need _and_ put the className on the outermost div?

Reply:
I could have the className on L61 take in one of many possible selectors:

```'filter-dropdown-text-field-container text-input'
'filter-dropdown-text-field-container date-input'
'filter-dropdown-text-field-container select-field'```

Reply:
.InstanceFieldInput .text-input-container {
}

Ideally, SUIT-friendly CSS would look like...
.InstanceFieldInput .ValueField .TextInput {
}